src/Slices/Resource/UI/ResourcesPage/Page.tsx

```diff
@@ -56,7 +56,7 @@ export const Page: React.FC = () => {
     if (RemoteData.isLoading(data)) return;
     setStaleData(data);
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [data.kind]);
+  }, [data.kind, JSON.stringify(data)]);
```

Comment:
isn't `data.kind` included in `JSON.stringify(data)`? so it could be removed? 
Also I don't know react enough to know if it is good practice to compare stringified data but I don't see why it would be bad :p

Reply:
Good point about `kind`
About stringify, it's one of the options for this problem, the data we can have here is relatively small so I think it should be ok. (An alternative could be a custom hook with deep comparison https://github.com/kentcdodds/use-deep-compare-effect)